src/components/MultiSelectCombobox/ComboboxItem.tsx

```diff
@@ -0,0 +1,19 @@
+import type { FC, MouseEventHandler } from 'react';
+import React from 'react';
+import DropdownItem from '../Dropdown/DropdownItem';
+
+interface ComboboxItemProps {
+  onClick: MouseEventHandler;
+}
+
+const ComboboxItem: FC<ComboboxItemProps> = ({ children, onClick }) => (
+  <DropdownItem
+    onClick={onClick}
+    className="overflow-hidden text-truncate"
+    title={children as string}
```

Comment:
Would it be a problem if `children` is passed in as a non-string react component? If it is, I think we should narrow down the `children` type for `ComboboxItemProps`

```diff
interface ComboboxItemProps {
  onClick: MouseEventHandler;
+ children: string;
}
```

Reply:
Good point. I'm actually going to narrow down a few of the other `children` types as well.